README.md

```diff
@@ -129,13 +143,15 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 * [AppCompatProcessor](https://github.com/mbevilacqua/appcompatprocessor) - AppCompatProcessor has been designed to extract additional value from enterprise-wide AppCompat / AmCache data beyond the classic stacking and grepping techniques.
 * [APT Hunter](https://github.com/ahmedkhlief/APT-Hunter) - APT-Hunter is Threat Hunting tool for windows event logs.
+* [Beagle](https://github.com/yampelo/beagle) - Incident response and digital forensics tool which transforms security logs and data into graphs.
```

Comment:
this one doesn't seem maintained anymore, remove it plz

Reply:
done with f7d79d80d81a96c63c43dde7555ef47ec235f748